app/controllers/application_controller.rb

```diff
@@ -47,6 +47,10 @@ def decorated_session
     @_decorated_session ||= DecoratedSession.new(sp: current_sp, view_context: view_context).call
   end
 
+  def signed_in_path
```

Comment:
oooh...hm yeah I tend not to make methods public *just* for testing. Seems problematic.